scripts/sysman/virtualFunctionManagement.yml

```diff
@@ -0,0 +1,252 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Virtual Function Management Properties"
+version: "1.9"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Virtual Function Management Extension Name"
+version: "1.9"
+name: $S_VIRTUAL_FUNCTION_MANAGEMENT_EXP_NAME
+value: '"$XS_extension_virtual_function_management"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual Function Management Extension Version(s)"
+version: "1.9"
+name: $s_vf_management_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual function memory type flags"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_info_mem_type_exp_t
+etors:
+    - name: MEM_TYPE_SYSTEM
+      value: "0"
+      desc: "System memory"
+    - name: MEM_TYPE_DEVICE
+      desc: "Device local memory"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual function utilization flag bit fields"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_info_util_exp_flag_t
+etors:
+    - name: INFO_NONE
+      value: "0"
+      desc: "No info associated with virtual function"
+    - name: INFO_MEM_CPU
+      value: "1"
+      desc: "System memory utilization associated with virtual function"
+    - name: INFO_MEM_GPU
+      desc: "Device memory utilization associated with virtual function"
+      value: "2"
+    - name: INFO_ENGINE
+      desc: 'Engine utilization associated with virtual function'
+      value: "4"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Virtual function management properties"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_exp_properties_t
+base: $s_base_properties_t
+members:
+    - type: $s_pci_address_t
+      name: "address"
+      desc: "[out] Virtual function BDF address"
+    - type: $s_uuid_t
+      name: uuid
+      desc: "[out] universal unique identifier of the device"
+    - type: $s_vf_info_util_flag_exp_t
+      name: "flag"
+      desc: "[out] Bit field of utilization flags available."
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Provides memory utilization values for a virtual function"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_util_mem_exp_t
+base: $s_base_state_t
+members:
+    - type: $s_vf_info_mem_type_exp_t
```

Comment:
Is this supposed to be a single type or a combination of flags? The `type` is named "type", but the `desc` says "flags".